hugegraph-core/src/main/java/com/baidu/hugegraph/backend/query/Query.java

```diff
@@ -343,6 +350,26 @@ public boolean paging() {
         return this.page != null;
     }
 
+    public void olap(boolean olap) {
+        this.olap = olap;
+    }
+
+    public boolean olap() {
+        return this.olap;
```

Comment:
can return olapPks != null && olapPks not empty

Reply:
olapPks is for query Vertex, olap is for index query